### ISSUE TITLE
How is your custom qemu performance?

### ISSUE THREAD

**Issue by Waterman178** [SinaKarvandi/Misc#1]
Can it trace the current instructions of the operating system, just like the run trace function of ollydbg?Does it support multi-core?If you do not enable vt acceleration, will it be slow to trace a certain software? But after vt is enabled, you cannot trace some instructions.

**Comment by SinaKarvandi** [comment]
Hello, yeah you can trace operating system instructions like olly but much faster. I don't know if it works on multi-core or not and also didn't test it with vt-x acceleration.

**Comment by Waterman178** [comment]
> Hello, yeah you can trace operating system instructions like olly but much faster. I don't know if it works on multi-core or not and also didn't test it with vt-x acceleration.

You try to start some software in qemu, such as Google Chrome or some games like League of legends, it should be very slow, do you know pyrebox and panda? They are both based on qemu, and single step tracking is very slow

**Comment by SinaKarvandi** [comment]
Yep, honestly, when I use it with other applications like Microsoft Edge, I realized that it's slow so I didn't use it anymore.
